dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/internal/SipContainerComponent.java

```diff
@@ -169,12 +172,14 @@ protected void deactivate(int reason) {
 	 */
 	@Modified
 	protected void modified(Map<String, Object> properties) {
-		if (c_logger.isTraceDebugEnabled())
-			c_logger.traceDebug("SipContainerComponent modified", properties);
-		PropertiesStore.getInstance().getProperties().updateProperties(properties);
-        useNetty = MetatypeUtils.parseBoolean(
-                "sipContainer", USE_NETTY,
-                properties.get(USE_NETTY), true);
+           if (c_logger.isTraceDebugEnabled())
+              c_logger.traceDebug("SipContainerComponent modified", properties);
+           PropertiesStore.getInstance().getProperties().updateProperties(properties);
+           useNetty = (!ProductInfo.getBetaEdition())?false:MetatypeUtils.parseBoolean(
+              "sipContainer", USE_NETTY,
+              properties.get(USE_NETTY), true);
+           if (c_logger.isTraceDebugEnabled() && useNetty)
+              c_logger.traceDebug("SipContainerComponent modified: useNetty=true");
 	}
```

Comment:
This is pretty hard to follow, what actually changed here? I think every line shows changed b/c of formatting differences tabs vs spaces etc. If we can clean that up it would be nice so we can more easily see what's changed at a glance.